library/src/main/resources/SAMLSpecRefMessage.properties

```diff
@@ -218,17 +215,17 @@ SAMLCore_3_3_4_c=If S2 [request subject] includes one or more <saml:SubjectConfi
   S1 [response subject] can be confirmed in the manner described by at least one \
   <saml:SubjectConfirmation> element in S2 [request subject].
 
-SAMLCore_3_4_1_1a=The special Format value urn:oasis:names:tc:SAML:2.0:nameid-format:encrypted \
+SAMLCore_3_4_1_1_a=The special Format value urn:oasis:names:tc:SAML:2.0:nameid-format:encrypted \
   indicates that the resulting assertion(s) MUST contain <EncryptedID> elements instead of \
   plaintext.
 
-SAMLCore_3_4_1_1b=When a Format defined in Section 8.3 other than \
+SAMLCore_3_4_1_1_b=When a Format defined in Section 8.3 other than \
   urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified or \
   urn:oasis:names:tc:SAML:2.0:nameid-format:encrypted is used, then if the identity provider \
   returns any assertions: the Format value of the <NameID> within the <Subject> of any <Assertion> \
   MUST be identical to the Format value supplied in the <NameIDPolicy>
 
-SAMLCore_3_4_1_1c=When a Format defined in Section 8.3 other than \
+SAMLCore_3_4_1_1_c=When a Format defined in Section 8.3 other than \
   urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified or \
   urn:oasis:names:tc:SAML:2.0:nameid-format:encrypted is used, then if the identity provider \
   returns any assertions: if SPNameQualifier is not omitted in <NameIDPolicy>, the SPNameQualifier \
```

Comment:
✏️ This message and its sibling (`SAMLCore_3_4_1_1_b`) are similar to `SAMLCore_8_3_6_a` and `SAMLCore_8_3_6_b` in that the two sibling messages each contain a common sentence or two and then an additional piece that's unique between the two. Should we make them consistent by putting the `...` in the two `SAMLCore_3_4_1_1` messages after the "returns any assertions" phrase?

Reply:
I decided to combine them.